src/components/core/DpBulkEditHeader.stories.tsx

```diff
@@ -0,0 +1,96 @@
+import type { Meta, StoryObj } from '@storybook/vue'
+import DpBulkEditHeader from './DpBulkEditHeader.vue'
+
+const meta: Meta<typeof DpBulkEditHeader> = {
+    component: DpBulkEditHeader,
+    title: 'Components/BulkEditHeader',
+    render: (args) => ({
+        components: {
+            DpBulkEditHeader
+        },
+        computed: {
+            selectedItemsCount () {
+                return args.options.filter(checked => checked.checked == true)
+            }
+        },
+        setup() {
+            const toggleSelectAll = () => {
+                for (let element of args.options) {
+                    if(element.checked === false) {
+                        element.checked = true
+                    } else {
+                        element.checked = false
+                    }
+                }
+            }
+
+            const resetSelection = () => {
+                for (const element of args.options) {
+                    element.checked = false
+                }
+            }
+
+            const deleteSelection = () => {
+                for (let element of args.options) {
+                    if(element.checked === true) {
+                        args.options = args.options.filter(checked => checked.checked != true)
+                    }
+                }
```

Comment:
As this is just an example, i would simply call `resetSelection()` here, instead of deleting the elements.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/615/commits/e2d7e2eefabf8df90725a47df489e4c9a2a92f16